desktop-widgets/CMakeLists.txt

```diff
@@ -55,7 +55,6 @@ set (SUBSURFACE_UI
 
 # the interface, in C++
 set(SUBSURFACE_INTERFACE
-	../map-widget/qmlmapwidgethelper.cpp
```

Comment:
shouldn't this be part of the last commit? Aren't we getting a compile error otherwise (haven't tried)

Reply:
I build and test every single commit, so there are no compile errors and it works.

I have no idea how CMake deals with the conflict, but please remember the old central CMake reference qmlmapwidgethelper.cpp, so this problem have existed for a long time.